views/contexts/model.go

```diff
@@ -194,6 +255,14 @@ func (m *Model) SetLoading(loading bool) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	m.loading = loading
+	// Write debug state when loading changes
+	go m.debugWriteLoadingState()
+}
+
+// debug write of loading state for diagnostics
+func (m *Model) debugWriteLoadingState() {
+	// best-effort write; don't fail on error
+	_ = os.WriteFile("/tmp/swarmcli_contexts_state_debug.json", []byte(fmt.Sprintf("{\"loading\":%v,\"count\":%d}\n", m.loading, len(m.contexts))), 0644)
 }
```

Comment:
Why do we not just log it?

Reply:
See https://github.com/mosonyi/swarmcli/pull/146/commits/2e0313785490b5c22e1a443f3675dfa9e542fac0